mrp_unbuild_subcontracting/models/stock_picking.py

```diff
@@ -56,14 +56,16 @@ def _action_done(self):
                 unbuild_ids_backorder = unbuilds_to_done.filtered(
                     lambda u: u.state == "draft"
                 ).ids
-            unbuilds_to_done.with_context(
-                subcontract_move_id=True, mo_ids_to_backorder=unbuild_ids_backorder
-            ).action_validate()
-            move = self.move_lines.filtered(lambda move: move.is_subcontract)
+            for unbuild in unbuilds_to_done:
+                unbuild.with_context(
+                    subcontract_move_id=True, mo_ids_to_backorder=unbuild_ids_backorder
+                ).action_validate()
+            moves = self.move_lines.filtered(lambda move: move.is_subcontract)
```

Comment:
I think it's more appropriate to make it like this.
```suggestion
            moves = picking.move_lines.filtered(lambda move: move.is_subcontract)
```